python/subunit/tests/test_test_protocol2.py

```diff
@@ -356,7 +356,7 @@ def test_packet_length_4_word_varint(self):
         packet_data = b'\xb3!@\xc0\x00\x11'
         self.check_events(packet_data, [
             self._event(test_id="subunit.parser", eof=True,
-                file_name="Packet data", file_bytes=packet_data,
+                file_name="Packet data", file_bytes=b'\xb3',
```

Comment:
Providing the actual failed packet still in this error case seems useful to me; If its super hard to preserve it I guess we can change the contract, but it will mean someone has to sit there with a protocol capture tool rather than just seeing what went wrong in a log file.

Reply:
[Took me a while](https://github.com/testing-cabal/subunit/pull/40#issuecomment-577667968) to figure out what was happening here, but done